.github/actions/spelling/excludes.txt

```diff
@@ -54,14 +54,26 @@ SUMS$
 \.zip$
 ^consolegit2gitfilters\.json$
 ^dep/
-^oss/
+^doc/reference/master-sequence-list.csv$
 ^doc/reference/UTF8-torture-test\.txt$
+^oss/
+^src/host/ft_uia/run\.bat$
+^src/host/runft\.bat$
+^src/host/runut\.bat$
```

Comment:
* Warning: src/host/ft_uia/run.bat: line 1, columns 1-1, Warning - Skipping `src/host/ft_uia/run.bat` because there seems to be more noise (7) than unique words (2) (total: 12 / 2). (noisy-file)
* Warning: src/host/runft.bat: line 1, columns 1-1, Warning - Skipping `src/host/runft.bat` because there seems to be more noise (6) than unique words (1) (total: 10 / 2). (noisy-file)
* Warning: src/host/runut.bat: line 1, columns 1-1, Warning - Skipping `src/host/runut.bat` because there seems to be more noise (4) than unique words (2) (total: 4 / 2). (noisy-file)